### ISSUE TITLE
Refinements to new tag subscription statistics page

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#5260]
Following up on #5224, we can tackle a few more things in this follow-up issue, now that @GettyOrawo has done a great job getting this page created!

* [ ] let's sort this DESCENDING or `desc` so we start with the most-used tags
* [ ] let's hide anything more than 25 results and show a "View all" link to show them?
* [ ] let's make the tag names into links that go to `/tag/TAGNAME`

and we can do more too! This is a great start!

**Comment by IshaGupta18** [comment]
Hey, @jywarren I think we will be tackling the first 2 tasks in the design suggested in #4603, so maybe the implementation will be a little different. What do you think? Also, I could start working on one or more of these subtasks soon, if that would be okay!

**Comment by jywarren** [comment]
yes, sounds good! You know this issue better than me, please go for it!

**Comment by divyabaid16** [comment]
Hey!!!
Can I work on the third one?

**Comment by jywarren** [comment]
We would love help!!!

**Comment by IshaGupta18** [comment]
@jywarren #5648 is my PR for the 1st subtask. Kindly have a look at it and let me know if its good to go or not. Thanks!

**Comment by jywarren** [comment]
Adding comments from https://github.com/publiclab/plots2/pull/5769 which I'm merging! For followups:

Please see these comments: https://github.com/publiclab/plots2/pull/5769#issuecomment-495235369

**Comment by jywarren** [comment]
Re-opening with some additional followups for sorting/organizing, all noted in these comments:

https://github.com/publiclab/plots2/pull/5769#issuecomment-495236681

We'd love more help with this!

**Comment by SidharthBansal** [comment]
@IshaGupta18 can you please help in breaking this issue into pieces? As many issues are already solved and you have worked on them since a long time, this will be great

**Comment by KarishmaVanwari** [comment]
@jywarren @IshaGupta18 Could you please clarify which issue is to be worked on now since it seems most have been already tackled?
